webapp/backend/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "Backend for NSMQ AI web application",
+  "main": "server.ts",
+  "scripts": {
+    "test": "npm run build && npm run jasmine",
+    "start": "ts-node-esm src/server.ts",
+    "tsc": "tsc",
+    "build": "npx tsc",
+    "prettier": "prettier --config .prettierrc \"src/**/**.ts\" --write",
+    "lint": "eslint \"src/**/**.ts\"",
+    "jasmine": "jasmine",
+    "watch": "tsc-watch --esModuleInterop src/server.ts --outDir ./build --onSuccess \"node ./build/server.js\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nsmq-ai/nsmqai.git"
+  },
+  "keywords": [
+    "nsmq"
+  ],
+  "author": "NSMQ Web app team",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/nsmq-ai/nsmqai/issues"
+  },
+  "homepage": "https://github.com/nsmq-ai/nsmqai#readme",
+  "devDependencies": {
+    "@types/jasmine": "^4.3.1",
+    "@types/supertest": "^2.0.12",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0",
+    "eslint": "^8.38.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "esm": "^3.2.25",
+    "jasmine": "^4.6.0",
+    "jasmine-spec-reporter": "^7.0.0",
+    "jasmine-ts": "^0.4.0",
+    "nodemon": "^2.0.22",
+    "prettier": "^2.8.7",
+    "supertest": "^6.3.3",
+    "ts-node": "^10.9.1",
+    "ts-node-dev": "^2.0.0",
+    "tsc-watch": "^6.0.0"
+  },
+  "dependencies": {
+    "@types/bcrypt": "^5.0.0",
+    "@types/cors": "^2.8.13",
+    "@types/express": "^4.17.17",
+    "bcrypt": "^5.1.0",
```

Comment:
not a major point but did you manually install bcrypt? just interested in it's function in your changes since it's more cryptographically used